### PR TITLE
refactor(frontend): migrate premium components

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by lukicenturi** [comment]
Probably not related to this PR, but we're missing space here 
image

**Comment by kelsos** [comment]
> Probably not related to this PR, but we're missing space here image

yes this is in components so I will have to fix there

File: frontend/app/src/components/defi/Borrowing.vue [hunk 1]
```diff
@@ -60,88 +59,80 @@ onMounted(async () => {
...
+            :value="summary.totalDebt"
+            show-currency="symbol"
+            fiat-currency="USD"
+          />
+        </StatCardColumn>
+      </template>
+    </StatCardWide>
+
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+      <VCard>
```
Comment (tewshi) [inline frontend/app/src/components/defi/Borrowing.vue:101]: can we use RuiCard here

**Review by tewshi — APPROVED** [review]
lgtm!

**Pull request merged**